recorder/recorder.conf

```diff
@@ -0,0 +1,11 @@
+[program:recorder]
+directory=/opt/app/recorder
+command=/opt/app/recorder/env/bin/python3 ./record.py
+autostart=true
+autorestart=true
```

Comment:
The Python script runs on a Compute Engine virtual machine. The startup script for the VM sets up [Supervisor](http://supervisord.org/), which uses this configuration to restart the Python script if it raises an Exception and exits.